heartbeat/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2.1'
+services:
+  beat:
+    build: ${PWD}/.
+    depends_on:
+    - proxy_dep
+    env_file:
+    - ${PWD}/build/test.env
+    environment:
+    - REDIS_HOST=redis
+    - REDIS_PORT=6379
+    - KIBANA_HOST=kibana
```

Comment:
Remove kibana

Reply:
Yeah, this are complicated now because they're split across this and #8491. This is already fixed there, let's close that one out and I'll rebase here.